tests/conftest.py

```diff
@@ -2,12 +2,8 @@
 
 import mockssh
 import pytest
-from git import Repo
```

Comment:
For some reason, removing this import causes `test_get_from_non_dvc_master` to fail, hence additional caplog.at_level there. I was unable to invesitage it in reasonable amount of time, so leaving this comment and creating issue.